edkrepo/commands/combo_command.py

```diff
@@ -47,13 +47,13 @@ def run_command(self, args, config):
             combo_list.append(manifest.general_config.current_combo)
         for combo in sorted(combo_list):
             if combo == manifest.general_config.current_combo:
-                print("* {}{}{}".format(Fore.GREEN, combo, Fore.RESET))
+                ui_functions.print_info_msg("current combo: " + combo, header=False)
             elif combo in combo_archive:
-                print("  {}{}{}{}".format(Fore.YELLOW, Style.BRIGHT, combo, Style.RESET_ALL))
+                ui_functions.print_info_msg("archived combo: " + combo, header=False)
             else:
-                print("  {}".format(combo))
+                ui_functions.print_info_msg(combo, header=False)
             if args.verbose:
                 sources = manifest.get_repo_sources(combo)
                 length = len(max([source.root for source in sources], key=len))
                 for source in sources:
-                    print("    {} : {}".format(source.root.ljust(length), source.branch))
+                    ui_functions.print_info_msg(source.root.ljust(length) + " : " +source.branch, header=False)

```

Comment:
Using the standard formatting is fine here. However please pre-pend the string with whitespace is is done in the original print.